packages/@react-types/list/src/index.d.ts

```diff
@@ -54,6 +55,8 @@ export interface SpectrumListProps<T> extends AriaListProps<T>, StyleProps, Spec
    * the collection's `selectionBehavior` prop and the interaction modality.
    */
   onAction?: (key: string) => void,
+  /** Whether `disabledKeys` applies to the whole row, or only selection. */
```

Comment:
I don't understand this documentation. Isn't a whole row the only thing selectable in a List or is selection all the selected items not just one row?

Reply:
Updated it to be slightly different. Better?

Reply:
yes, thanks!